### ISSUE TITLE
Compendium Errors

### ISSUE THREAD

**Issue by Keledrath** [Sheep-y/trpg-dnd-4e-db#44]
Hello, I adore the project you've done. My 4e group has been running a Living campaign for the past 6ish years, and we've got a fairly extensive list of compendium errors we've located and corrected for our offline version. I figured I would share them with you in case you wanted to implement them here
https://docs.google.com/document/d/1yhKzsuMKoWImvvWaVRXJatlrYNvA_nWwKBRso93qAks/edit

**Comment by mw-stanley** [comment]
I don't think this is going to get resolved because this tool is not responsible for the content of the online compendium - which does contain errors, you're correct - but rather just uncritically downloads whatever content is available there. 

If the online compendium were available, I'd suggest submitting a bug report to its maintainers. Since the online compendium is now gone, this project is probably defunct anyway.

If you do want to obtain an updated compendium with most of these errors corrected, there are other projects that do maintain the compendium content.

**Comment by Keledrath** [comment]
I thought the same, which was why I hadn't posted anything before now, but I was reading the documentation and found the "Fixed errors" section, which specifically had a link to here for errors that were not fixed.

**Comment by Buic** [comment]
Could you perhaps share that corrected offline copy somewhere? I've got a fairly long game going as well but don't really have the time to do such a thing myself.

**Comment by Keledrath** [comment]
Sure https://www.dropbox.com/s/3cejrsmxiuxix1q/Portable%20Compendium%20New.zip?dl=0
Our admin usually patches it up every 6 months or so with any new errors we found

**Comment by Buic** [comment]
Thanks!
